packages/common/src/groups/_internal/GroupBusinessRules.ts

```diff
@@ -113,10 +113,10 @@ export const GroupPermissionPolicies: IPermissionPolicy[] = [
   },
   {
     permission: "hub:group:workspace:content",
-    dependencies: ["hub:group:edit"],
+    dependencies: ["hub:group:view"],
```

Comment:
Let's double check with Thomas on this change.

Reply:
This is for the entity group content, not the entity content :)

Reply:
Ahhhh my bad. Thanks for the clarification!